documentation/Readme.md

```diff
@@ -0,0 +1,47 @@
+# Project Title
+- Cool logo
+
+---
+
+### Introduction
+
+- Summary of the purpose and function of this project
```

Comment:
I do not understand how the words "purpose" and "function" used here are different.

Reply:
Good call. Rephrased it with better explanation.